src/release/breaking-changes/window-singleton.md

```diff
@@ -0,0 +1,378 @@
+  tester.platformDispatcher.localesTestValue = <Locale>[const Locale('en-us'), const Locale('ar-jo')];
+  tester.platformDispatcher.localeTestValue = const Locale('ar-jo');
+  tester.platformDispatcher.nativeSpellCheckServiceDefinedTestValue = false;
+  tester.platformDispatcher.platformBrightnessTestValue = Brightness.dark;
+  tester.platformDispatcher.semanticsEnabledTestValue = true;
+  tester.platformDispatcher.textScaleFactorTestValue = 2.0;
+});
+```
+
+#### Resetting platform-specific properties
+
+Similarly to setting properties, resetting platform-specific properties consists
+mainly of changing from the `binding.window` accessor to the
+`platformDispatcher` accessor.
+
+##### Resetting individual properties
+
+Code before migration:
+
+```dart
+testWidgets('test name', (WidgetTester tester) async {
+  addTeardown(tester.binding.window.clearAccessibilityFeaturesTestValue);
+  addTeardown(tester.binding.window.clearAlwaysUse24HourFormatTestValue);
+  addTeardown(tester.binding.window.clearBrieflyShowPasswordTestValue);
+  addTeardown(tester.binding.window.clearDefaultRouteNameTestValue);
+  addTeardown(tester.binding.window.clearInitialLifecycleStateTestValue);
+  addTeardown(tester.binding.window.clearLocalesTestValue);
+  addTeardown(tester.binding.window.clearLocaleTestValue);
+  addTeardown(tester.binding.window.clearNativeSpellCheckServiceDefinedTestValue);
+  addTeardown(tester.binding.window.clearPlatformBrightnessTestValue);
+  addTeardown(tester.binding.window.clearSemanticsEnabledTestValue);
+  addTeardown(tester.binding.window.clearTextScaleFactorTestValue);
+});
+```
+
+Code after migration:
+
+```dart
+testWidgets('test name', (WidgetTester tester) async {
+  addTeardown(tester.platformDispatcher.clearAccessibilityFeaturesTestValue);
+  addTeardown(tester.platformDispatcher.clearAlwaysUse24HourFormatTestValue);
+  addTeardown(tester.platformDispatcher.clearBrieflyShowPasswordTestValue);
+  addTeardown(tester.platformDispatcher.clearDefaultRouteNameTestValue);
+  addTeardown(tester.platformDispatcher.clearInitialLifecycleStateTestValue);
+  addTeardown(tester.platformDispatcher.clearLocalesTestValue);
+  addTeardown(tester.platformDispatcher.clearLocaleTestValue);
+  addTeardown(tester.platformDispatcher.clearNativeSpellCheckServiceDefinedTestValue);
+  addTeardown(tester.platformDispatcher.clearPlatformBrightnessTestValue);
+  addTeardown(tester.platformDispatcher.clearSemanticsEnabledTestValue);
+  addTeardown(tester.platformDispatcher.clearTextScaleFactorTestValue);
+});
+```
+
+##### Resetting all properties at once
+
+Code before migration:
+
+```dart
+testWidgets('test name', (WidgetTester tester) async {
+  addTeardown(tester.binding.window.clearAllTestValues);
+});
+```
+
+Code after migration:
+
+```dart
+testWidgets('test name', (WidgetTester tester) async {
+  addTeardown(tester.platformDispatcher.clearAllTestValues);
+});
+```
+
+## Timeline
+
+Landed in version: TODO<br>
```

Comment:
Reminder to myself to fill this in before submitting this guide. The change is expected to land sometime today or tomorrow.